src/Notifications/NotificationsUIViewModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Windows;
+using System.Windows.Media;
+using Dynamo.ViewModels;
+
+namespace Dynamo.Notifications
+{
+    class NotificationsUIViewModel : ViewModelBase
+    {
+        public PointCollection TooltipPointerPoints { get; set; }
+
+        //Direction of the shadow shown in the Popup
+        public double ShadowTooltipDirection { get; set; } = 90;
+
+        //This value will describe the needed space for drawing pointers inside the Popup
+        public int PopupBordersOffSet { get; set; } = 15;
+
+        /// <summary>
+        /// Popup Width, by default it will have an exaggerated Width to accommodate localized languages
+        /// TODO it should really be responsive either using grid of auto canvas:
+        /// https://stackoverflow.com/questions/855334/wpf-how-to-make-canvas-auto-resize
+        /// </summary>
+        public double PopupRectangleWidth { get; set; } = 310;
+
+        /// <summary>
+        /// Popup Height, by default it will have a Height of 568
+        /// </summary>
+        public double PopupRectangleHeight { get; set; } = 568;
+                
+        double PointerWidth = 25;
```

Comment:
Just to confirm, are all of the above properties need to be public? And this one will be internal?

Reply:
yes, thats right